src/views/certifications/views/[slug]/components/program-hero/program-hero.module.css

```diff
@@ -0,0 +1,26 @@
+.heroBackground {
+	background: var(--background-image);
+	background-position: top center;
+	background-repeat: no-repeat;
+	background-size: cover;
+	bottom: -10%;
+	filter: blur(8px);
```

Comment:
We use small dimension images and blur them rather than exporting the blurred gradient images at full size. This lets us greatly reduce the file size of the images. Note: this might change in a future PR, pending ongoing work to refine the hero appearance on the design side.

Reply:
ooooo ✨ v smart 👏🏻